packages/edit-widgets/src/components/header/style.scss

```diff
@@ -6,6 +6,36 @@
 	padding: 0 $grid-unit-20;
 }
 
+.edit-widgets-header-toolbar {
+	display: inline-flex;
+	align-items: center;
```

Comment:
We don't need the `display: inline-flex` here because it's already set by the `components-accessible-toolbar` class. 

Any reason for adding `align-items: center`? We shouldn't need it either, because the buttons are all the same height so will align by default.

Reply:
Yeah I mainly just copied paste from the code in `edit-post` 😅 , I'll re-examinate them! Thx!